modules/cluster-logging-systemd-scaling.adoc

```diff
@@ -68,59 +67,69 @@ For more information on systemd settings, see link:https://www.freedesktop.org/s
 +
 // Defaults from https://github.com/openshift/openshift-ansible/pull/3753/files#diff-40b7a7231e77d95ca6009dc9bcc0f470R33-R34
 
-. Convert the `journal.conf` file to base64:
+. Convert the `journal.conf` file to base64 and store it in a variable (`jrnl_cnf`) by running the following command:
```

Comment:
Do you mind replacing the parens in this instructional text?  They don't seem imperative and the ISG has the following text in section Punctuation > Parentheses around p97.

> Use parentheses to identify items such as abbreviations, symbols, and measurements, but avoid using
parentheses in general text.

If you are open to suggestion, one way might be "...and store it in a variable that is named `jrnl_cnf` by..."

Reply:
Looks like those changes got messed up during squash. Will restore it again.